scenarios/crowdsecurity/http-sensitive-files.yaml

```diff
@@ -0,0 +1,17 @@
+type: leaky
+format: 2.0
+debug: true
+name: crowdsecurity/http-sensitive-files
+description: "Detect attempt to access to sensitive files (.log, .db ..) or folders (.git)"
+filter: 'evt.Meta.service == "http" and any(File("sensitive_data.txt"), { evt.Parsed.request endsWith #})'
+groupby: "evt.Meta.source_ip"
+data:
+   - source_url: https://raw.githubusercontent.com/crowdsecurity/sec-lists/master/web/sensitive_data.txt
+   - dest_file: sensitive_data.txt
+capacity: 2
```

Comment:
I think we want to distinct on the URI to limit the impact of false positives :)

Reply:
umm, I not sure, if we add `distinct` and an attacker try to access at least 3 different (files and directories in the list) it's just no ?

Reply:
After reflection, you're right.